apidoc/Titanium/Android/Activity.yml

```diff
@@ -193,6 +193,12 @@ methods:
         type: Titanium.Android.Intent
         optional: true
 
+  - name: setSupportActionBar
+    summary: Sets a toolbar instance to be used as an ActionBar.
+    parameters:
+      - name: toolbar
+        type: Titanium.UI.Android.Toolbar  
```

Comment:
The setter/getter are generated by the apidoc-generator automatically, you don't need to specify it.